en/identity-server/next/docs/apis/restapis/authenticators.yaml

```diff
@@ -242,6 +409,92 @@ components:
         self:
           type: string
           example: /t/carbon.super/api/server/v1/configs/authenticators/eDUwOUNlcnRpZmljYXRlQXV0aGVudGljYXRvcg
+    UserDefinedLocalAuthenticatorCreation:
+      description: This represents the configuration for creating the user defined local authenticator.
```

Comment:
configuration for creating the user defined local authenticator -> configuration to create a user defined local authenticator